.github/workflows/ci.yml

```diff
@@ -149,9 +149,14 @@ jobs:
           path: massa-as-sdk
           ref: main
       - name: "Use the current branch as dependency for gas-calibration"
-        run: cd gas-calibration && sed -i 's/main/${{ github.head_ref || github.ref_name }}/g' Cargo.toml && sed -i 's!massa-sc-runtime = { git = "https://github.com/massalabs/massa-sc-runtime", features = \["gas_calibration"\] }!'"$(cat ../massa-execution-worker/Cargo.toml | grep massa-sc-runtime | sed 's!/!\\/!g' | sed 's!}!, features = ["gas_calibration"]}!g' )"'!g' Cargo.toml && cat Cargo.toml
+        run: > 
+          cd gas-calibration && 
+          sed -i 's/main/${{ github.head_ref || github.ref_name }}/g' Cargo.toml && 
+          sed -i 's!massa-sc-runtime = { git = "https://github.com/massalabs/massa-sc-runtime", features = \["gas_calibration"\] }!'"$(cat ../massa-execution-worker/Cargo.toml | grep massa-sc-runtime | sed 's!/!\\/!g' | sed 's!}!, features = ["gas_calibration"]}!g' )"'!g' Cargo.toml
```

Comment:
Still not sure to understand this line :-/ 
Can it be simplified / doc ?

Reply:
I just added documentation. It difficult for me to simplify I don't know different ways to do it in bash